lib/scoped_search/auto_complete_builder.rb

```diff
@@ -208,7 +208,7 @@ def complete_value
         .uniq
         .map(&field.field)
         .compact
-        .map { |v| v.to_s =~ /\s/ ? "\"#{v}\"" : v }
+        .map { |v| v.to_s =~ /\s/ ? "\"#{v.gsub('"', '\"')}\"" : v }
```

Comment:
The only one that comes to mind is AR's quoting for table/column names etc, but since the tokenizer is in scoped_search itself and only supports double quotes, I think gsub's going to be best.